backend/site-api/src/app/site/update_site.rs

```diff
@@ -56,9 +57,23 @@ pub async fn update_site(
         validator.validate_pages(value)?;
     }
     let has_update_key = dto.update_key.is_some();
+
+    // Update `content_updated_at` if any of `defaults`, `context`, or `pages` changes.
+    let mut content_updated_at: Option<i64> = None;
+
+    if dto.defaults.is_some() || dto.context.is_some() || dto.pages.is_some() {
```

Comment:
Since I fixed the frontend to only send updated data, we can avoid the comparison here and just check if the fields are present in the DTO.